### PR TITLE
fix(directive): skip validation for empty values

### PR DESCRIPTION
**Pull request by lepipele**

### CODE REVIEW

File: ng-intl-tel-input.directive.spec.js [hunk 1]
```diff
@@ -47,6 +47,13 @@ describe('ng-intl-tel-input', function () {
     expect(form.tel.$valid).toBe(true);
   });
 
+  it('should set the field as valid with empty input', function () {
```
Comment (SyntaxStacks) [inline ng-intl-tel-input.directive.spec.js]: cacf9f86392afeac7d83fdd2291b53af

**Comment by SyntaxStacks** [comment]
Thanks for the contribution @lepipele !

**Comment by lepipele** [comment]
@SyntaxStacks Thanks for helping with merging ;)

One question - can I subscribe somewhere to get notified when bower package gets updated so that I can pull new version with this change?

**Comment by SyntaxStacks** [comment]
I'm working on that now, scroll up to the top and click watch, maybe even a star if you'd like ;)

**Comment by lepipele** [comment]
I was just curious if there is way to subscribe to bower package notifications... I guess I'll Google it and star/watch on GitHub for now ;) Thanks!

**Comment by SyntaxStacks** [comment]
yo @lepipele I bumped the version to 1.1.0
https://www.npmjs.com/package/ng-intl-tel-input

**Comment by lepipele** [comment]
@SyntaxStacks Thanks for letting me know man! ;)

**Pull request merged**